index.js

```diff
@@ -228,6 +227,9 @@ exports.extract = function (cwd, opts) {
       return next()
     }
 
+    header = map(header) || header
```

Comment:
This assigment will override the previous `header.name` assigment, it should be done after this.